workspaces/adr/plugins/adr/src/alpha.tsx

```diff
@@ -50,8 +58,46 @@ export const adrSearchResultListItemExtension =
     },
   });
 
+/** @alpha */
+export const adrEntityContentExtension = EntityContentBlueprint.make({
+  name: 'entity',
+  params: {
+    defaultPath: '/adrs',
+    defaultTitle: 'ADRs',
+    filter: 'kind:component',
```

Comment:
Sorry to contradict @awanlin but I don't think we need to filter this to components only - for instance we use ADRs for system kinds as well internally

Reply:
As the owner, I'm cool with the push back 👍 

If I'm not mistaken this would be the default which you can override in the config. I made the assumption that generally the default of having this filtered to Component was reasonable. Without a filter I'm pretty sure this would show on Users and Groups.

Reply:
on second thought I'm fine with this as a default since consumers can just override this

Reply:
yup what you said sounds reasonable @awanlin - apologies I'm still familiarizing myself with the new frontend system :)

Reply:
No worries at all!